test/realm_test.g.dart

```diff
@@ -0,0 +1,141 @@
+// GENERATED CODE - DO NOT MODIFY BY HAND
+
+part of 'realm_test.dart';
+
+// **************************************************************************
+// RealmObjectGenerator
+// **************************************************************************
+
+class Car extends _Car with RealmObject {
+  static var _defaultsSet = false;
+
+  Car({
+    String? make,
+  }) {
+    if (make != null) _make = make;
+    _defaultsSet = _defaultsSet ||
+        RealmObject.setDefaults<Car>({
+          'make': "Tesla",
+        });
+  }
+
+  Car._();
+
+  @override
+  String get make => RealmObject.get<String>(this, 'make') as String;
+  set _make(String value) => RealmObject.set(this, 'make', value);
+
+  static SchemaObject get schema => _schema ??= _initSchema();
+  static SchemaObject? _schema;
```

Comment:
given `Car.schema` will be use very rarely is there a value in caching the SchemaObject in memory? Should we generate a getter instead as in the `realm_tests.gen.dart`

Reply:
Really just used as a flag to indicate if `_initSchema` needs to be called

Reply:
oh, I missed this is a static field. Initially I misread it thinking this will be an instance field which would increase the required memory for every single instance. 
scratch my comment.